dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -104,11 +104,11 @@ public static void main(String[] arguments)
     if (dashboardArguments.hasVersionlessCoordinates()) {
       generateAllVersions(dashboardArguments.getVersionlessCoordinates());
     } else {
-      Path output =
-          dashboardArguments.hasFile()
-              ? generate(dashboardArguments.getBomFile())
-              : generate(dashboardArguments.getBomCoordinates());
-      System.out.println("Wrote dashboard into " + output.toAbsolutePath());
+      if (dashboardArguments.hasFile()) {
```

Comment:
What about using `else if` here? It's simpler.

```java
if (getVersionlessCoordinates()) {
  generate...
} else if (hasFile()) {
  generate...
} else {
  generate...
}
```

Reply:
That's more clear. Updated.